### PR TITLE
Removes thermals from off-ship roles.

### PR DESCRIPTION
**Pull request by Kaksisilma**
This PR DOES NOT TOUCH MERC, NINJA OR REGULAR TRAITORS
This PR DOES NOT TOUCH MERC, NINJA OR REGULAR TRAITORS
This PR DOES NOT TOUCH MERC, NINJA OR REGULAR TRAITORS
This PR DOES NOT TOUCH MERC, NINJA OR REGULAR TRAITORS

This PR removes the access to thermals from Vox, Ascent, Skrell and Infantry RIGs, and adds Night Vision modules in their place.

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/suits/merc.dm [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 	initial_modules = list(
 		/obj/item/rig_module/mounted/lcannon,
-		/obj/item/rig_module/vision/thermal,
+		/obj/item/rig_module/vision/nvg,
```
Comment (Kaksisilma) [inline code/modules/clothing/spacesuits/rig/suits/merc.dm:29]: Mercs can still buy Thermals through the uplink. This removal here is intended to force them to spend TC.

**Comment by Kaksisilma** [comment]
I am not sure if the infantry rig is the one I have touched, but I would like to ask your opinion on whether infantry should have thermals or not before this is merged.

@Anton-Kr 
@ProbablyCarl

**Comment by ProbablyCarl** [comment]
USE TGM AAAAAAAAAAAAAAAAAAAAAAAA

**Comment by Kaksisilma** [comment]
Fuck

**Comment by Anton-Kr** [comment]
@Kaksisilma Just remove the vox map files. I'll make @Armyguy implement it along with his changes/fixes PR

**Comment by Kaksisilma** [comment]
Right I fucked this up. One second.

**Pull request closed without merging**